radio/src/telemetry/spektrum.cpp

```diff
@@ -724,6 +740,30 @@ void processSpektrumPacket(const uint8_t *packet)
       }
     } // I2C_GPS_BIN
 
+    else if (i2cAddress == I2C_FP_BATT) {
+      flightPackTelemetry = true;
+      // Lemon-RX G2: No Bat2: Current (-1.0 A)
+      if (sensor->startByte == 6 && ((int16_t) value) == -10) continue;  
+    } // I2C_FP_BATT
+
+    else if (i2cAddress == I2C_PBOX) {
+      // hide mAh Consumption already reported in Fligh Pack message 
+      // No Bat Voltage, hide Voltage and Consumption
+      if (flightPackTelemetry && (sensor->startByte == 4 ||  sensor->startByte == 6)) {
+          continue;
+      else if ((sensor->startByte == 0 || sensor->startByte == 4) && // Bat1 Voltage
+          spektrumGetValue(packet + 4, 0, uint16) == 0) continue;
+      else if ((sensor->startByte == 2 || sensor->startByte == 6) && // Bat2 Voltage
+          spektrumGetValue(packet + 4, 2, uint16) == 0) continue;
```

Comment:
Can youi please fix the indentation here and add curly brackets for each "if".
This is almost impossible to read if you do not already know what it does already.